libs/markets/src/lib/components/market-info/tooltip-mapping.tsx

```diff
@@ -106,4 +106,43 @@ export const tooltipMapping: Record<string, ReactNode> = {
   insurancePoolFraction: t(
     'The fraction of the insurance pool balance that is carried over from the parent market to the successor.'
   ),
+  commitmentMinimumTimeFraction: t(
+    `This is a market parameter. It specifies the minimum fraction of time LPs must spend 'on the book' providing their committed liquidity.`
+  ),
+  feeCalculationTimeStep: t(
+    'This is a market parameter. This is providers fee calculation time step. Specifies how often the quality of liquidity supplied by the LPs is evaluated and fees arising from that period are earmarked for specific parties.'
+  ),
+  performanceHysteresisEpochs: t(
+    'This is a market parameter. Specifies the number of epochs over which past performance will continue to affect rewards.'
+  ),
+  SLACompetitionFactor: t(
+    `This is a market parameter. The SLA completion factor. Specifies the maximum fraction of their accrued fees an LP that meets the SLA implied by market.liquidity.commitmentMinTimeFraction will lose to liquidity providers that achieved a higher SLA performance than them. `
+  ),
+  bondPenaltyParameter: t(
+    'This is a network parameter. It is used to calculate the penalty to liquidity providers when they cannot support their open position with the assets in their margin and general accounts. Valid values: any decimal number >= 0 with a default value of 0.1.'
+  ),
+  nonPerformanceBondPenaltySlope: t(
+    'This is a network parameter. It is used to calculate by how much the LP bond is slashed if an LP fails to reach the minimum SLA. Valid values: any decimal number >= 0 with a default value of 2.0.'
```

Comment:
I'll add the formula for this to the docs, a link to that would probably help here. I can tell you the prospective link or it can go into a future PR, up to you

Reply:
I am happy to do it in a future PR once we have that link